app/models/device.model.ts

```diff
@@ -0,0 +1,19 @@
+import sequelize from '../../config/sequelize';
+import * as Sequelize from 'sequelize';
+
+const Device = sequelize.define('tb_client_devices', {
+  UUID: {
+    type: Sequelize.STRING(255),
+    allowNull: false
+  },
+  client_id: {
```

Comment:
We need to add user_id there too, as wallet has `client_id` and `user_id`. After adding `user_id`, add the check to find by `user_id` in the controller methods.

Reply:
You mean, a separate method to retrieve devices by `user_id` provided, correct?